pkg/controller/submariner/submariner_controller.go

```diff
@@ -509,6 +509,8 @@ func newEnginePodTemplate(cr *submopv1a1.Submariner) corev1.PodTemplateSpec {
 			Volumes: []corev1.Volume{
 				{Name: "host-slash", VolumeSource: corev1.VolumeSource{HostPath: &corev1.HostPathVolumeSource{Path: "/"}}},
 			},
+			// The gateway engine must be able to run on any flagged node, regardless of existing taints
```

Comment:
@mangelajo I *think* this also makes the Engine and Routeagent pods to be scheduled on the Master nodes. Can you please check once.

Reply:
Yes correct, it's intended. There is also no other way (that I know) to workaround random taints.